main.py

```diff
@@ -17,25 +17,7 @@
 
     sessionmaker = config_db(config.database_url)
 
-    bot_repository = PostgresBotRepository(sessionmaker)
-
-    bot_service = BotServiceV1(bot_repository)
-
-    bot_controller = BotControllerV1(bot_service)
-
     app = FastAPI()
-
-    app.add_api_route(
-        "/bots",
-        endpoint=bot_controller.fetch_chatbots,
-        status_code=status.HTTP_200_OK,
-        response_model=list[Bot],
-    )
-    app.add_api_route(
-        "/bots",
-        endpoint=bot_controller.create_chatbot,
-        methods=["POST"],
-        status_code=status.HTTP_201_CREATED,
-    )
+    add_bot_routes(app, sessionmaker)

```

Comment:
Why declare the dependency injection logic in the bot module? Initially, everything was declared in main so the dependencies were made explicit in one single place.